src/lib/forms/core/ObjectListField.js

```diff
@@ -64,7 +63,7 @@ export class ObjectListField extends Component {
             onClick={() => this.onItemClick(null, items.length)}
           >
             <List.Content>
-              <Icon name="add" />
+              <Icon name="id card" />
```

Comment:
not blocking for now: as the field is meant to be the generic one (ObjectListfield), the icon ideally should be passed as a prop, to change it depending on the type of the object